scripts/generator/graphql_generator/csharp/SDK/NativeMessage.cs.erb

```diff
@@ -25,8 +23,9 @@ namespace <%= namespace %>.SDK {
         }
 
         public void Respond(string message) {
+            #if UNITY_IOS
```

Comment:
Should this be `#if !SHOPIFY_MONO_UNIT_TEST`? 
Wouldn't this 💥 , now that you've added `UNITY_IOS` to the defined flags in mcs